packages/block-library/src/quote/edit.js

```diff
@@ -3,6 +3,7 @@
  */
 import { __ } from '@wordpress/i18n';
 import { AlignmentToolbar, BlockControls, RichText } from '@wordpress/block-editor';
+import { BlockQuotation } from '@wordpress/components';
```

Comment:
I wonder if we should gather all these Platform specific components in a separate package or otherwise under the same parent variable. 

Whether it's `import { Blockquote } from '@wordpress/platform'` 

or 

```js
import { PlatformComponents } from '@wordpress/components';
const { Blockquote } = PlatformComponents;
````

Reply:
Yep, this is a good point. I think we should extract all those components into their own package and maybe re-export from `@wordpress/components` for backward compatibility only. It's definitely out of the scope of this PR @SergioEstevao. Anyway, it's something we should decide on quite soon as you will need to use more HTML tags like:
- `figure`
- `ul`, `ol`, `li`
- `img`
- `div`